### PR TITLE
Fix Favicon - PyCon India Blog

### LINKED ISSUE #122: [Bug] Favicon for blog not changed
The favicon in use for the blog hasn't been changed since it was applied in 2017, as Pycon India 2018 has a new logo (looks pretty good). We should use the official logo to maintain uniformity throughout the websites. 

- [x]  Replace favicon and delete old unused media related to it. 

Also, it could be one of the cases, where developers like to say. 
"This is not a bug, but a feature." :laughing:

### PR DESCRIPTION
**Pull request by vipulgupta2048**
This pull request fixes #122 
- updated the favicon.png for PyCon India 2018 from PyCon India 2017

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 *.pyc
 .idea/
 *.pid
+.venv
```
Comment (realslimshanky) [inline .gitignore]: Please add a new line after this.

**Comment by vipulgupta2048** [comment]
I am not sure why I added a mention of `.venv` in the .gitignore. Removed it, thanks for mentioning it

**Comment by ananyo2012** [comment]
The favicon you updated is not the updated one. This one is the logo now 
logo

**Comment by vipulgupta2048** [comment]
@ananyo2012 Updated the favicon.png as mentioned. Here's a preview of the same running on my local server.
screenshot_2018-04-21_10-09-41
Although it's too small to be visible.

**Pull request merged**